addons/hr_holidays/models/hr_leave.py

```diff
@@ -1631,6 +1631,12 @@ def activity_update(self):
                             'res_id': holiday.id,
                             'res_model_id': self.env.ref('hr_holidays.model_hr_leave').id,
                         })
+            elif holiday.state == 'validate1':
+                holiday.activity_feedback(['hr_holidays.mail_act_leave_approval'])
+                holiday.activity_schedule(
+                    'hr_holidays.mail_act_leave_second_approval',
+                    note=note,
+                    user_id=holiday.sudo()._get_responsible_for_approval().id or self.env.user.id)
```

Comment:
simply adding this code back will not work, if you scroll a bit up (line 1624) you will see that `_get_responsible_for_approval` might return multiple users and then this line will crash. We need to adapt this code as well. For inspiration, you can check this commit - https://github.com/odoo/odoo/pull/142327/commits/c5a3abbe65ab567b105aa4776def24c8e1927ee2#diff-38469def2f870bb866f971f57797dd7c21b6a95d52a8eae72f832f0eea2434f9R1505-R1520

Reply:
@sofiagvaladze Thank you for your feedback! I adapted the code similarly to the referenced commit.

Reply:
@sofiagvaladze Thank you for your guidance! I think I got the commit to be working the same as yours for 17.0. Can you double-check?